tests/System.IO.Compression.Tests/BrotliPrimitivesTests.cs

```diff
@@ -131,6 +133,34 @@ public void WrongWindowSize(uint windowSize)
 
         [Theory]
         [InlineData(1)]
+        [InlineData(10)]
+        [InlineData(1024)]
+        public void DecompressRandomInvalidData(int totalSize)
+        {
+            Brotli.State state = new Brotli.State();
+            byte[] data = new byte[totalSize];
+            new Random(42).NextBytes(data);
+            Assert.Throws<System.IO.IOException>(delegate { Brotli.Decompress(data, Array.Empty<byte>(), out int consumed, out int written, ref state);  });
```

Comment:
It's just way to transform call to System.Action and handle and check exception. I'm really interested in other ways to do it